tests/test_models.py

```diff
@@ -3,7 +3,7 @@
 import numpy as np
 import numpy.testing as npt
 import os
-
```

Comment:
this is fine

Reply:
According to the course content, you're meant to have more tests in this file. What you should have is:
Standard deviation for one patient with multiple observations.
Standard deviation for two patients.
Graph includes a standard deviation graph.
Standard deviation function should raise an error if given empty data.
Computing standard deviation where deviation is different from variance.
Standard deviation function should give correct result given negative inputs.
Function should work with numpy arrays